catalog.html

```diff
@@ -0,0 +1,147 @@
+    <li class="h-client"><a class="link-active" href="">клиенты</a></li>
+    <li class="h-store"><a class="link-active active" href="catalog.html">магазин</a></li>
+    <li class="h-kontakt"><a class="link-active" href="">контакты</a></li>
+    <li class="h-basket"><a class="link-active" href=""><img src="img/cart-icon.svg" alt="cart-icon">корзина</a>
+    </li>
+  </ul>
+  <div class="m-heading">
+    <div class="gallery-m"><h1>Магазин готовых шаблонов</h1></div>
+  </div>
+</header>
+<main>
+  <div class="main-catalog">
+    <div class="left-column">
+      <form class="slide-conteiner" action="catalog.html" method="get">
+        <div class="c-custom">
+          <div class="custom-h3"><h3>стоимость:</h3></div>
+          <div class="range-filter">
+            <div class="range-controls">
+              <div class="scale">
+                <div class="bar"></div>
+              </div>
+              <div class="toggle toggle-min"></div>
+              <div class="toggle toggle-max"></div>
+            </div>
+            <div class="price-controls">
+              <label class="min-price">от <input name="min-price" type="text" value="0"></label>
+              <label class="max-price">до <input name="max-price" type="text" value="15000"></label>
+            </div>
+          </div>
+        </div>
+        <div class="radio-conainer">
+          <h3>сетка:</h3>
+          <label><input type="radio" name="option1" checked>адаптивная</label>
+          <label><input type="radio" name="option1">фиксированая</label>
+          <label><input type="radio" name="option1">резиновая</label>
+        </div>
+        <div class="chekbox-container">
+          <h3>особенности:</h3>
+          <label><input type="checkbox" name="slider">слайдер</label>
+          <label><input type="checkbox" name="block-advant" id="block-advant">блок преимуществ</label>
+          <label><input type="checkbox" name="news">новости</label>
+          <label><input type="checkbox" name="gallery">галлерея</label>
+          <label><input type="checkbox" name="basket">корзина</label>
+          <button class="range-button">показать</button>
+        </div>
+      </form>
+    </div>
+    <div class="right-column">
+      <div class="sort">
+        <div class="heading-sort"><h3>сортировать:</h3></div>
+        <div>по цене</div>
+        <div>по типу</div>
+        <div>по названию</div>
+        <div class="sort-button">
+          <form>
+          <input type="submit" name="up" class="up" style="background-image: url(img/icon-down-dir.svg);" />
+          <button class="down"></button>
+          </form>
+        </div>
+      </div>
+      <ul class="c-gallery">
+        <li><a href="">
+          <div class="browser"><img src="img/browser.svg"></div>
+          <div class="browser-wind"><h3>седона</h3>информационный сайт для туристов
+            <button class="wind-b">9900р</button>
+          </div>
+          <img src="img/img-sedona.jpg" alt="maket-page"></a></li>
```

Comment:
Исправить форматирование
```suggestion
          <img src="img/img-sedona.jpg" alt="maket-page">
        </a></li>
```